src/tempo.py

```diff
@@ -62,6 +62,7 @@ class Tempo:
     def __init__(
         self,
         container: ops.Container,
+        charm: Optional[object] = None,
```

Comment:
Would it be possible to pass just the ingress object? That could make it less dependent on having an object named exactly `ingress` in the charm.